src/ado2gh/GithubApiFactory.cs

```diff
@@ -19,7 +19,14 @@ public virtual GithubApi Create()
         {
             var githubPat = _environmentVariableProvider.GithubPersonalAccessToken();
             var githubClient = new GithubClient(_octoLogger, _client, githubPat);
-            return new GithubApi(githubClient);
+            return new GithubApi(githubClient, "https://api.github.com");
```

Comment:
minor thing, but couldn't these 3 lines be simplified to just:

`return Create("https://api.github.com")`